DQM/TrackingMonitorSource/python/TrackingDataMCValidation_Standalone_cff.py

```diff
@@ -1,44 +1,223 @@
+                             hltPathsOfInterest = cms.vstring("HLT_ZeroBias_v"),
+                             triggerResults = cms.untracked.InputTag("TriggerResults","","HLT"),
+                             triggerEvent = cms.untracked.InputTag("hltTriggerSummaryAOD","","HLT")
+                             )
+
+# HLT path selector Muon
+hltPathFilterMuon = cms.EDFilter("HLTPathSelector",
+                             processName = cms.string("HLT"),
+                             verbose = cms.untracked.bool(False),
+                             hltPathsOfInterest = cms.vstring("HLT_IsoMu24_v"),
+                             triggerResults = cms.untracked.InputTag("TriggerResults","","HLT"),
+                             triggerEvent = cms.untracked.InputTag("hltTriggerSummaryAOD","","HLT")
+                             )
+
+# HLT path selector Electron
+hltPathFilterElectron = cms.EDFilter("HLTPathSelector",
+                             processName = cms.string("HLT"),
+                             verbose = cms.untracked.bool(False),
+                             hltPathsOfInterest = cms.vstring("HLT_Ele32_WPTight_Gsf_v"),
+                             triggerResults = cms.untracked.InputTag("TriggerResults","","HLT"),
+                             triggerEvent = cms.untracked.InputTag("hltTriggerSummaryAOD","","HLT")
+                             )
+
+# HLT path selector ttbar
+hltPathFilterTtbar = cms.EDFilter("HLTPathSelector",
+                             processName = cms.string("HLT"),
+                             verbose = cms.untracked.bool(False),
+                             hltPathsOfInterest = cms.vstring("HLT_Ele32_WPTight_Gsf_v","HLT_IsoMu24_v"),
+                             triggerResults = cms.untracked.InputTag("TriggerResults","","HLT"),
+                             triggerEvent = cms.untracked.InputTag("hltTriggerSummaryAOD","","HLT")
+                             )
+
+# Z->MuMu event selector
 ztoMMEventSelector = cms.EDFilter("ZtoMMEventSelector")
+muonTracks = cms.EDProducer("ZtoMMMuonTrackProducer")
+# Z->ee event selector
 ztoEEEventSelector = cms.EDFilter("ZtoEEEventSelector")
+electronTracks = cms.EDProducer("ZtoEEElectronTrackProducer")
+#ttbar event selector
+ttbarEventSelector = cms.EDFilter("ttbarEventSelector")
+ttbarTracks = cms.EDProducer("TtbarTrackProducer")
+
+# Added module for V0Monitoring for Ks only
+# KshortMonitor = v0Monitor.clone()
+# KshortMonitor.FolderName = cms.string("Tracking/V0Monitoring/Ks")
+# KshortMonitor.v0         = cms.InputTag('generalV0Candidates:Kshort')
+# KshortMonitor.histoPSet.massPSet = cms.PSet(
+#   nbins = cms.int32 ( 100 ),
+#   xmin  = cms.double( 0.400),
+#   xmax  = cms.double( 0.600),
+# )
+
+# For MinBias
+standaloneTrackMonitorMC = standaloneTrackMonitor.clone(
+    puScaleFactorFile = cms.untracked.string("PileupScaleFactor_316060_wrt_nVertex_ZeroBias.root"),
+    doPUCorrection    = cms.untracked.bool(True),
+    isMC              = cms.untracked.bool(True)
```

Comment:
```suggestion
    puScaleFactorFile = "PileupScaleFactor_316060_wrt_nVertex_ZeroBias.root",
    doPUCorrection    = True,
    isMC             =True
```
and remove explicit type declaration also from the other cloned modules (see e.g point 4 in https://twiki.cern.ch/twiki/bin/view/CMS/OpenRecoTasks#Review_and_optimization_of_the_p)